angular.json

```diff
@@ -210,6 +231,22 @@
               "extractLicenses": false,
               "sourceMap": true,
               "namedChunks": true
+            },
+            "logica-open": {
+              "fileReplacements": [
+                {
+                  "replace": "projects/cardiovascular-health/src/environments/environment.ts",
+                  "with": "projects/cardiovascular-health/src/environments/environment.logica-open.ts"
+                }
+              ]
+            },
+            "mock": {
+              "fileReplacements": [
+                {
+                  "replace": "projects/cardiovascular-health/src/environments/environment.ts",
+                  "with": "projects/cardiovascular-health/src/environments/environment.mock.ts"
+                }
+              ]
             }
```

Comment:
please remove the `mock` configuration because it does not work with the cardio app